ntbs-service/Models/Entities/Alerts/DataQualityChildECMLevel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Linq.Expressions;
+using ntbs_service.Helpers;
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.Models.Entities.Alerts
+
+{
+    public class DataQualityChildECMLevel : Alert
+    {
+        private const int MinNumberDaysDraftForAlert = 45;
+
+        public static readonly Expression<Func<Notification, bool>> NotificationQualifiesExpression =
+            n => n.NotificationStatus == NotificationStatus.Notified
+                 && n.CreationDate < DateTime.Now.AddDays(-MinNumberDaysDraftForAlert)
+                 && (n.PatientDetails.Dob.Value.Month * 100 + n.PatientDetails.Dob.Value.Day > n.NotificationDate.Value.Month * 100 + n.NotificationDate.Value.Day
+                     ? (n.NotificationDate.Value.Year - n.PatientDetails.Dob.Value.Year - 1) < 16
+                     : n.NotificationDate.Value.Year - n.PatientDetails.Dob.Value.Year < 16)
```

Comment:
Is LINQ really too dumb to convert `n.NotificationDate.Value < n.PatientDetails.Dob.Value.AddYears(16)`?

Reply:
That seems to work fine, not sure what I was trying before!